images/linux/scripts/installers/yq.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/install.sh
+
+YQ_BINARY=yq_linux_amd64
+
+# As per https://github.com/mikefarah/yq#wget
+YQ_URL=$(curl -s https://api.github.com/repos/mikefarah/yq/releases/latest | jq -r ".assets[].browser_download_url | select(endswith(\"$YQ_BINARY.tar.gz\"))")
+echo "Downloading latest yq from $YQ_URL"
+
+download_with_retries "$YQ_URL" "/tmp" "${YQ_BINARY}.tar.gz"
+tar xzf "/tmp/${YQ_BINARY}.tar.gz" -C "/tmp"
+mv /tmp/${YQ_BINARY} /usr/local/bin/yq
```

Comment:
Assuming this is the ubuntu:latest? If someone installs yq previously under /usr/bin using some previous version then this would break things.

Reply:
Hi @skonto! 
No, it wasn't made intentionally, we were not aware of typical yq usage. Is it usually installed in /usr/bin rather than /usr/local/bin?

Reply:
Hi thanks for the quick response! It depends on how users setup things. Afaik `/usr/bin` is for OS level installations, `/usr/local/bin` for user supplied binaries (allows overriding system binaries). I think this change (PR) was intended to be at the VM/OS level. Btw we have such [a setup](https://github.com/openshift-knative/serverless-operator/blob/034e1eb83e0170677f32d794fc95cf311ac007f3/.github/workflows/validate.yaml#L31). WDYTH?

Reply:
Well, this setup won't override yq installed in `/usr/local/bin` 😩
I'll prepare a PR with moving yq to /usr/bin, but the new image with the fix will be deployed next week. Could you please add a workaround until that? Something like ` if [[ $(which yq) =~ "/usr/local/bin" ]]; then  sudo rm $(which yq); fi`
Sorry for the inconvenience!

Reply:
Hmm apologies for this, looking at some other tool installation scripts, `/usr/local/bin` seemed to be the standard practice, but I agree `/usr/bin` probably makes more sense here...

Reply:
No problem, yeah no strong opinions thanks for the response @Yann-J :)